### ISSUE TITLE
Instructor copying course: Progress bar does not load when no feedback sessions are copied 

### ISSUE THREAD

**Issue by FergusMok** [TEAMMATES/teammates#11911]
Master Branch & V8.18.0 

**Steps to reproduce**

https://user-images.githubusercontent.com/10182564/180641338-93540c15-066f-4227-9e77-4d6ac50191db.mp4
1. Login as instructor
2. Attempt to copy course, but do not select any feedback sessions to copy

**Expected behaviour**
Loading bar to progress when copying, show the correct progress, and close when it is finished. Show the successful alert when it is done.

**Actual behaviour**
As shown in the video, the bar doesn't progress at all, and doesn't close when it is done.

**Comment by damithc** [comment]
@FergusMok thanks for reporting this. I could not reproduce it in the release branch (tried only once though). Is it a recent regression?

**Comment by FergusMok** [comment]
@damithc I realized it may be because I did not select any feedback sessions to copy, and you might have, and that'll result in the difference in behavior. This might not be a regression, as there's no changes to the code since it was implemented 

![image](https://user-images.githubusercontent.com/10182564/180652521-360ca29a-6a87-4dc0-bddf-6fec5b94117e.png)

For others who are interested in trying to solve the issue:
https://github.com/TEAMMATES/teammates/blob/46979fb6f3b1752e47c0c0bf6984e34939265051/src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts#L321-L333

It's because the promise here is only resolved once it's done copying all the sessions. However, if there's no sessions to copy, the promise is never resolved, and the progress bar is not updated.
